client/api/src/client.rs

```diff
@@ -139,13 +143,15 @@ pub struct UsageInfo {
 impl fmt::Display for UsageInfo {
 	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
 		write!(f,
-			"caches: ({} state, {} db overlay), i/o: ({} tx, {} write, {} read, {} tx size)",
+			"caches: ({} state, {} db overlay), i/o: ({} tx, {} write, {} read, {} avg tx, {}/{} state cache read / total)",
```

Comment:
I really don't understand this output :sweat_smile: 

The first caches prints the size? In bytes? Maybe show in mb or something? At least append `b` or whatever to know what it is.

And this newly added state cache reads and state reads is also not understandable if you not read the code.

Reply:
Those are key cache reads to total reads, this is just technical trace
but I will try improve on output, thanks!

Reply:
Updated